tutorials/io/70_reading_eyetracking_data.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+r"""
+.. _tut-importing-eyetracking-data:
+
+=======================================
+Importing Data from Eyetracking devices
+=======================================
+
+Eyetracking devices record a persons point of gaze, usually in relation to a
+screen. Typically, gaze position (also referred to as eye or pupil position)
+and pupil size are recorded as separate channels. This section describes how to
+read data from supported eyetracking manufacturers.
+
+MNE-Python provides functions for reading eyetracking data. When possible,
+MNE-Python will internally convert and store eyetracking data according to an
+SI unit (for example radians for position data, and meters for pupil size).
+
+.. seealso:: Some MNE functions may not be available to eyetracking and other
+             physiological data, because MNE does not consider them to be data
+             channels. See the :doc:`glossary </glossary>` for more
+             information.
+
+.. _import-eyelink_asc:
+
+SR Research (Eyelink) (.asc)
+============================
+
+.. note:: MNE-Python currently only supports reading Eyelink eyetracking data
+          stored in the ASCII (.asc) format.
+
+Eyelink recordings are stored in the Eyelink Data Format (EDF; .edf), which are
+binary files and thus relatively complex to support. To make the data in EDF
+files accessible, Eyelink provides the application EDF2ASC, which converts EDF
+files to a plain text ASCII format (.asc). These files can be imported
+into MNE using :func:`mne.io.read_raw_eyelink`.
+
+.. note:: The Eyelink Data Format (EDF), should not be confused
+          with the European Data Format, the common EEG data format that also
+          uses the .edf extension.
+
+Supported measurement types from Eyelink files include eye position, pupil
+size, saccadic velocity, resolution, and head position (for recordings
+collected in remote mode). Eyelink files often report occular events (blinks,
+saccades, and fixations), MNE will store these events as `mne.Annotations`.
+For More information on the various measurement types that can be present in
+Eyelink files, read below.
+
+Eye Position Data
+-----------------
+
+Eyelink samples can report eye position data in pixels, units of visual
+degrees, or as raw pupil coordinates. Samples are written as (x, y) coordinate
+pairs (or two pairs for binocular data). The type of position data present in
+an ASCII file will be detected automatically by MNE. The three types of
+position data are explained below.
+
+Gaze
+^^^^
+Gaze position data report the estimated (x, y) pixel coordinates of the
+participants's gaze on the stimulus screen, compensating for head position
+changes and distance from  the screen. This datatype may be preferable if you
+are interested in knowing where the participant was looking at on the stimulus
+screen. The default pixel range in Eyelink systems is that of a 1024 x 768
+screen, with (0, 0) at the top left.
```

Comment:
doesnt this depend on your setup? we have different stimulus screens..
its definitely minor, but i'd only keep the default ref location "The default  (0, 0) location for Eyelink systems is at the top left of the screen."

Reply:
see [9157d98](https://github.com/mne-tools/mne-python/pull/11152/commits/9157d98b59c9ee403ed6995c05e42a3fa55ab773)